src/RouteCollector.php

```diff
@@ -164,9 +164,13 @@ private function checkForDuplicateRoute(string $path, array $methods = null) : v
         });
 
         if (! empty($matches)) {
-            throw new Exception\DuplicateRouteException(
-                'Duplicate route detected; same name or path, and one or more HTTP methods intersect'
-            );
+            $match = reset( $matches );
```

Comment:
167 | ERROR | [x] Space after opening parenthesis of function call prohibited
 167 | ERROR | [x] Expected 0 spaces before closing bracket; 1 found

Remove the space around `$matches`